Assets/Scripts/UIManager.cs

```diff
@@ -23,17 +23,34 @@ public class UIManager : MonoBehaviour
     [SerializeField]
     private Button _restartLevelButton;
 
+    [SerializeField]
+    private Button _restartButton;
+
+    [SerializeField]
+    private Button _premadeLevelButton;
+
+    [SerializeField]
+    private Image _optionsMenuBackground;
+
+    [SerializeField]
+    private ShapesManager _shapesManager;
+
+    private bool _isOptionsMenuOpen = false;
+
     void Start()
     {
         _winText.gameObject.SetActive(false);
         _loseText.gameObject.SetActive(false);
         _nextLevelButton.gameObject.SetActive(false);
         _restartLevelButton.gameObject.SetActive(false);
+        _restartButton.gameObject.SetActive(false);
+        _premadeLevelButton.gameObject.SetActive(false);
+        _optionsMenuBackground.gameObject.SetActive(false);
     }
 
     void Update()
```

Comment:
if you're not gonna use this, might as well delete it. we try to keep empty methods out of the codebase to keep it cleaner.

Reply:
I'm currently using the void Start Method to set these gameObjects to inactive

Reply:
yeah I meant the `Update` method, it's empty